src/components/common/headers/loggedInHeader/index.tsx

```diff
@@ -1,20 +1,85 @@
-import { Button } from "antd";
+import React from "react";
+import { Button, Layout, Menu, Dropdown } from "antd";
 import { useDispatch } from "react-redux";
-
+import Link from "next/link";
 import { logout } from "../../../../network/services/auth";
 import { AppDispatch } from "../../../../store";
+import styles from "../siteHeader.module.scss";
+import SearchSection from "../../search/search";
+import LoginModal from "../../../componentPages/Login/loginModal";
+import RegistrationModal from "../../../componentPages/registration/registrationModal";
 
+const { Header } = Layout;
 const LoggedInHeader = () => {
   const dispatch = useDispatch<AppDispatch>();
 
   const logOut = async () => {
     await dispatch(logout());
   };
-
+  const onClick = ({ key }) => {
+    if (key == 3) {
+      logOut();
+    }
+  };
+  const menu = (
+    <Menu onClick={onClick} >
+      <Menu.Item key="0">
+        <Link href="/settings">Account Settings</Link>
+      </Menu.Item>
+      <Menu.Item key="1">
+        <a >Support Camps</a>
+      </Menu.Item>
+      <Menu.Divider />
+      <Menu.Item key="3" >Log Out</Menu.Item>
+    </Menu>
+  );
   return (
-    <h1 style={{ backgroundColor: "#a5ec00" }}>
-      Header for logged-in users <Button onClick={logOut}>Log out</Button>
-    </h1>
+    <React.Fragment>
+      <Header className={styles.wrap}>
+        <div className={styles.logo}>
+          <img src={"/images/logo.svg"} alt="Logo" />
+        </div>
+        <nav className={styles.navWrap}>
+          <Menu mode="horizontal" className={styles.nav}>
```

Comment:
Probably a new component for Menu as well

Reply:
Done